README.md

```diff
@@ -3,60 +3,70 @@
 
-The second RBAC file which can be found at [here](https://raw.githubusercontent.com/galasa-dev/helm/release/charts/ecosystem/rbac-admin.yaml) allows someone to do the helm install/upgrade/delete commands, this will need to be modified slightly to include authorised users.
 
-To install a Galasa Ecosystem using Helm, use the following command (after adding the repo detailed above) :-
+### Installation
+To install a Galasa Ecosystem using Helm, use the following command (after adding the repo detailed above). Note: The Galasa Ecosystem Helm chart will deploy three persistent volumes. If you need to provide a Kubernetes storage class for these PVs, you can override the `storageClass` value as shown in the command below. If you are deploying to minikube, you can use the `standard` storage class created for you by minikube, but this is not required.
 
-    helm install --set galasaVersion=0.23.0 --set externalHostname=my.host.name <release-name> galasa/ecosystem --wait 
+```
+helm install [--set storageClass=mystorageclass] --set galasaVersion=0.23.0 --set externalHostname=my.host.name <release-name> galasa/ecosystem --wait 
+``` 
 
-It is very important that the `--wait` is included as the chart uses a post-install hook to complete the installation.  During the installation, you will see the engine-controller, metrics and resource-monitor pods restart a few times, this is expected as they cannot start until the api pod has correct initialised and the api pod cannot start until the etcd and couchdb have completed their startups. 
+To install the latest development version of the Galasa Ecosystem chart, replace `galasa/ecosystem` in the above command with the path to the [`ecosystem`](./charts/ecosystem) directory in this repository.
 
-The `galasaVersion` value is the version of Galasa you want to run with.  You should not use latest to ensure each pod in the Ecosystem is running at the same level.
+The `--wait` flag ensures the chart installation has completed before marking it as "Deployed". During the installation, the API pod waits for the etcd and RAS pods to initialise while the engine-controller, metrics, and resource-monitor pods wait for the API pod to initialise.
 
-The `externalHostname` value is the DNS hostname or IP address of the Kubernetes node that be used to access the Galasa NodePort services.
+The `galasaVersion` value is the version of Galasa you want to run. You should not use latest to ensure each pod in the Ecosystem is running at the same level.
 
-The Galasa Ecosystem Helm chart will deploy three persistent volumes.  If you need to provide a Kubernetes storage class for these PVs, then please use the following command:-
+The `externalHostname` value is the DNS hostname or IP address of the Kubernetes node that be used to access the Galasa NodePort services. If you are deploying to minikube, this can be retrieved by running `minikube ip`.
```

Comment:
> If you are deploying to minikube, this can be retrieved by running `minikube ip`.

Maybe put the minikube stuff in a call-out separely ?

Reply:
Addressed in https://github.com/galasa-dev/helm/pull/1/commits/fad3b94c4581df0dae003f50aac19569f88aa2f1